src/main/webapp/app/shared/confirm-entity-name/confirm-entity-name.component.ts

```diff
@@ -89,4 +89,11 @@ export class ConfirmEntityNameComponent implements OnInit, OnDestroy, ControlVal
     registerOnValidatorChange(fn: () => void) {
         this.onValidatorChange = fn;
     }
+
+    private compareWithEntityName(control: FormControl): ValidationErrors | null {
```

Comment:
we prefer `undefined` over `null` in the client code. Could you change or is null necessary due to the Validators API?

Reply:
`null` is here indeed the specified return type:
https://angular.io/guide/form-validation#validator-functions

```
constructor(controls: TControl, validatorOrOpts?: ValidatorFn | ValidatorFn[] | AbstractControlOptions | null, asyncValidator?: AsyncValidatorFn | AsyncValidatorFn[] | null);
    controls: ɵTypedOrUntyped<TControl, TControl, {
        [key: string]: AbstractControl<any>;
    }>;
```